tools/run_go_bin_builder.py

```diff
@@ -84,34 +69,28 @@ def main():
     os.mkdir(noms_output_dir)
     assert os.path.exists(noms_output_dir)
 
-    cmd_path = '%s:%s' % (os.getenv('PATH'), get_go_install_path())
-
-    platform_pattern = re.compile(r'^(?P<os>\w+)-(?P<arch>\w+)$')
-    for osarch in PLATFORMS:
-        platform_match = platform_pattern.match(osarch)
-        assert platform_match
-
+    for osname, cpuarch in PLATFORMS:
+        osarch = '%s-%s' % (osname, cpuarch)
         platform_output_dir = os.path.join(noms_output_dir, osarch)
         os.mkdir(platform_output_dir)
         assert os.path.exists(platform_output_dir)
 
         env = copy.copy(os.environ)
         env.update({
-            'GOOS': platform_match.group('os'),
-            'GOARCH': platform_match.group('arch'),
+            'GOOS': osname,
+            'GOARCH': cpuarch,
             'GOPATH': workspace,
-            'PATH': cmd_path,
```

Comment:
I think the PATH stuff was right though? It was using an absolute path to the binary that was unnecessary.